putt/settings.py

```diff
@@ -39,9 +39,13 @@
     'django.contrib.staticfiles',
     'rest_framework',
     'rest_framework_simplejwt',
+    'drf_spectacular',
+    'corsheaders',
+    'projects',
```

Comment:
`projects ` should be included like `users.apps.UsersConfig`,  and the order had to change to from above.

Reply:
resolved